test/powershell/Modules/Microsoft.PowerShell.Management/Alias.Tests.ps1

```diff
@@ -0,0 +1,51 @@
+Describe "Basic Valdiation of Alias Provider" -Tags "CI" {
```

Comment:
Fixed spelling of validation.
I talked to @JamesWTruher and he said to put these provider tests in Modules/Microsoft.PowerShell.Management/.